Assets/MixedRealityToolkit/Providers/Hands/BaseHandVisualizer.cs

```diff
@@ -92,7 +92,7 @@ void IMixedRealityHandJointHandler.OnHandJointsUpdated(InputEventData<IDictionar
                 return;
             }
 
-            MixedRealityHandTrackingProfile handTrackingProfile = MixedRealityToolkit.Instance.ActiveProfile.InputSystemProfile.HandTrackingProfile;
+            MixedRealityHandTrackingProfile handTrackingProfile = InputSystem?.InputSystemProfile?.HandTrackingProfile;
```

Comment:
>InputSystem [](start = 66, length = 11)

Curious - why does this check for null but the ones down below dont?

Reply:
good question, i can make them match